consensus/quorum/one-node-one-vote.go

```diff
@@ -120,10 +120,12 @@ func (v *uniformVoteWeight) AmIMemberOfCommitee() bool {
 	}
 	identity, _ := pubKeyFunc()
 	everyone := v.DumpParticipants()
-	myVoterID := identity.SerializeToHexStr()
```

Comment:
AmIMemberOfCommitee us called for every message received. Better make it efficient.

Reply:
you mean store the serialized hex str also part of multi-bls struct for public key?

Reply:
It's a double for loop. If len(everyone) = 250. It's very expensive.  Better just use map lookup. At least avoid nested loop

Reply:
This code won't be used after staking, instead `func (v *stakedVoteWeight) AmIMemberOfCommitee() bool ` which is already made efficient. Do we still need to update this code?